client/my-sites/plan-features/header.jsx

```diff
@@ -103,7 +104,12 @@ export class PlanFeaturesHeader extends Component {
 					{ planLevelsMatch( selectedPlan, planType ) && (
 						<PlanPill>{ translate( 'Suggested' ) }</PlanPill>
 					) }
-					{ popular && ! selectedPlan && <PlanPill>{ translate( 'Popular' ) }</PlanPill> }
+					{ popular && ! selectedPlan && ! isEligibleForPlanStepTest && (
+						<PlanPill>{ translate( 'Popular' ) }</PlanPill>
+					) }
+					{ popular && ! selectedPlan && isEligibleForPlanStepTest && (
+						<PlanPill>{ translate( 'Most popular' ) }</PlanPill>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **38** times:
`translate( 'Most Popular Content' )` *ES Score: 7*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/plans-page-test-features)